std_msgs/Header header
float32 tail_amp
float32 tail_freq
float32 tail_bias
float32 height_goal
float32 tilt_goal
bool tail_enable
